config.toml.sample

```diff
@@ -37,6 +37,28 @@ credit_contract = "osmo1f2m24wktq0sw3c0lexlg7fv4kngwyttvzws3a3r3al9ld2s2pvds87jq
 multiplier = 1
 denom = "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2"
 
+[protocols.neutron-astroport]
+rpc = "https://rpc-kralum.neutron-1.neutron.org:443/"
+source = "astroport"
+jitter = 10000
+concurrency_limit = 6
+pagination_limit = 30
+[protocols.neutron-astroport.assets.dATOM]
+multiplier = 1
+denom = "ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9"
+pair_contract = "neutron15d0se9nw9stqx9ms9ey85gfpraqwl80t5urmjfk2jwtqxpxk3x3qctjt4n"
+
+[protocols.neutron-astroport-generator]
+rpc = "https://rpc-kralum.neutron-1.neutron.org:443/"
+source = "generator"
+jitter = 10000
+pagination_limit = 50
+generator_contract = "neutron173fd8wpfzyqnfnpwq2zhtgdstujrjz2wkprkjfr6gqg4gknctjyq6m3tch"
+[protocols.neutron-astroport-generator.assets.dATOM]
+multiplier = 1
+denom = "ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9"
+pair_contract = "neutron15d0se9nw9stqx9ms9ey85gfpraqwl80t5urmjfk2jwtqxpxk3x3qctjt4n"
```

Comment:
Is this contract address really related to the asset? If not, it should be extracted from `assets` section to the main protocol section

Reply:
I mean is this `pair_contract` will differ from asset to asset or not

Reply:
I think this field should actually be an array... There may be many different pair contracts for a single asset. And yes, this field is related to the asset. My proposed solution will look like this:

```toml
[protocols.abc.assets.xyz]
multiplier = 1
denom = "ibc/12345"
pair_contract = [ "neutron1abc", "neutron1bcd" ]
```

Reply:
fine, in this case, how to decide which contract to choose from array?

Reply:
All of them in a loop. Imagine if asset of interest is ATOM and we monitor two pairs: ATOM/NTRN and ATOM/TIA. ATOM/NTRN will have its own pair_contract and ATOM/TIA will have its own one.

Reply:
@ratik what do you think